### PR TITLE
Updated Existing Ammo to 1.6

### PR DESCRIPTION
**Pull request by N7Huntsman**
I still need to add the new ammo types to the rifle calibers--I'll take care of that tomorrow.

### CODE REVIEW

File: Defs/Ammo/Grenade/14.5x114mmSoviet.xml [hunk 1]
```diff
@@ -0,0 +1,307 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```
Comment (NoImageAvailable) [inline Defs/Ammo/Grenade/14.5x114mmSoviet.xml]: Wrong folder, this needs to be in HighCaliber, not Grenade

File: Defs/Ammo/Grenade/50BMG.xml [hunk 2]
```diff
@@ -0,0 +1,311 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```
Comment (NoImageAvailable) [inline Defs/Ammo/Grenade/50BMG.xml]: Wrong folder as well

File: Defs/Ammo/HighCaliber/14.5x114mmSoviet.xml [hunk 3]
```diff
@@ -164,30 +187,65 @@
...
+    </products>
+    <workAmount>10400</workAmount>
   </RecipeDef>
 
   <RecipeDef ParentName="AmmoRecipeBase">
     <defName>MakeAmmo_145x114mm_HE</defName>
-    <label>make 14.5x114mm (HE) cartridge x200</label>
-    <description>Craft 200 14.5x114mm (HE) cartridges.</description>
-    <jobString>Making 14.5x114mm (HE) cartridges.</jobString>
+    <label>make 14.5x114mm (AP-HE) cartridge x200</label>
```
Comment (NoImageAvailable) [inline Defs/Ammo/HighCaliber/14.5x114mmSoviet.xml]: This should just be HE

File: Defs/ThingDefs_Misc/Weapons_RangedNeolithic.xml [hunk 4]
```diff
@@ -90,16 +90,6 @@
 				<compClass>CompEquippable</compClass>
 			</li>
 		</comps>
-		<recipeMaker>
```
[the remark this answers is not in the document]
Comment (N7Huntsman) [reply Defs/ThingDefs_Misc/Weapons_RangedNeolithic.xml]: We discussed the pila, and unless I'm mistaken, we reached the conclusion that given how low the work cost was for them when crafted individually, it made more sense to craft them in batches, like most ammo. The pila itself still had the recipeMaker from vanilla, which handles it like a weapon.

File: Defs/Ammo/HighCaliber/14.5x114mmSoviet.xml (hunk 3 above)
Comment (N7Huntsman) [reply Defs/Ammo/HighCaliber/14.5x114mmSoviet.xml]: Fixed. Was going by Stat Sheet ammo type for labels.

File: Defs/Ammo/30x29mmGrenade.xml [hunk 5]
```diff
@@ -0,0 +1,215 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```
Comment (NoImageAvailable) [inline Defs/Ammo/30x29mmGrenade.xml]: Wrong folder

File: Defs/Ammo/40x46mmGrenade.xml [hunk 6]
```diff
@@ -0,0 +1,214 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```
Comment (NoImageAvailable) [inline Defs/Ammo/40x46mmGrenade.xml]: Wrong folder

File: Defs/Ammo/Rifle/303British.xml [hunk 7]
```diff
@@ -73,12 +76,55 @@
...
+    <statBases>
+      <MarketValue>0.15</MarketValue>
+    </statBases>
+    <ammoClass>IncendiaryAP</ammoClass>
+    <cookOffProjectile>Bullet_303British_Incendiary</cookOffProjectile>
+  </ThingDef>
+
+  <ThingDef Class="CombatExtended.AmmoDef" ParentName="303BritishBase">
+    <defName>Ammo_303British_HE</defName>
+    <label>.303 British cartridge (AP-HE)</label>
```
Comment (NoImageAvailable) [inline Defs/Ammo/Rifle/303British.xml]: Should be HE not AP-HE

File: Defs/Ammo/Rifle/303British.xml [hunk 8]
```diff
@@ -120,12 +164,55 @@
...
+			  <def>Flame_Secondary</def>
+			  <amount>8</amount>
+			</li>
+		  </secondaryDamage>
+		</projectile>
+	  </ThingDef>
+	  
+	  <ThingDef Class="CombatExtended.AmmoDef" ParentName="Base303BritishBullet">
+		<defName>Bullet_303British_HE</defName>
+		<label>.303 British bullet (AP-HE)</label>
```
Comment (NoImageAvailable) [inline Defs/Ammo/Rifle/303British.xml]: Also HE

**Pull request merged**